pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/maler/redigerbar/InnvilgelseAvAlderspensjonTrygdeavtaleDto.kt

```diff
@@ -0,0 +1,95 @@
+package no.nav.pensjon.brev.api.model.maler.redigerbar
+
+import no.nav.pensjon.brev.api.model.AlderspensjonRegelverkType
+import no.nav.pensjon.brev.api.model.Sakstype
+import no.nav.pensjon.brev.api.model.maler.BrevbakerBrevdata
+import no.nav.pensjon.brev.api.model.maler.RedigerbarBrevdata
+import no.nav.pensjon.brev.api.model.vedlegg.DineRettigheterOgMulighetTilAaKlageDto
+import no.nav.pensjon.brev.api.model.vedlegg.MaanedligPensjonFoerSkattAP2025Dto
+import no.nav.pensjon.brev.api.model.vedlegg.MaanedligPensjonFoerSkattDto
+import no.nav.pensjon.brevbaker.api.model.DisplayText
+import no.nav.pensjon.brevbaker.api.model.Kroner
+import java.time.LocalDate
+
+
+@Suppress("unused")
+data class InnvilgelseAvAlderspensjonTrygdeavtaleDto(
+    override val pesysData: PesysData,
+    override val saksbehandlerValg: SaksbehandlerValg,
+) : RedigerbarBrevdata<InnvilgelseAvAlderspensjonTrygdeavtaleDto.SaksbehandlerValg, InnvilgelseAvAlderspensjonTrygdeavtaleDto.PesysData> {
+
+    data class SaksbehandlerValg(
+        @DisplayText("Tittel - Ny beregning av innvilget alderspensjon. Ingen endring av uttaksgraden")
+        val nyBeregningAvInnvilgetAP: Boolean,
+        @DisplayText("Tittel - Innvilgelse av alderspensjon eller økt uttaksgrad")
+        val innvilgelseAPellerOektUttaksgrad: Boolean,
+        @DisplayText("Slutthandling medfører: Ingen endring i pensjonen")
+        val ingenEndringIPensjonen: Boolean,
```

Comment:
Her også - ingen endring her, og under brytarar for hhv økning og reduksjon. Trur det er betre å halde det til så få brytarar som mogleg.